src/views/MentorDashboard.js

```diff
@@ -96,12 +96,29 @@ export default function MentorDashboard() {
           redirect: "follow",
         };
 
-        fetch(`${STATS_API}/stats/mentors`, requestOptions)
-          .then((response) => response.text())
-          .then((result) => {
-            setStudents(JSON.parse(result)["students"]);
-          })
-          .catch((error) => console.log("error", error));
+        // fetch(`${STATS_API}/stats/mentors`, requestOptions)
+        //   .then((response) => response.text())
+        //   .then((result) => {
+        //     setStudents(JSON.parse(result)["students"]);
+        //   })
+        //   .catch((error) => console.log("error", error));
+
+        // const config = {
+        //   headers: {
+        //     Bearer: `${localStorage.getItem("mentor_jwt")}`
+        //   },
+        // };
+
+        // axios
+        //   .post(`${BACKEND_URL}/stats/mentor/${localStorage.getItem("mentor_username")}`, {},  config)
+        //   .then((res) => {
+        //     console.log(res);
+        //     // setStats(res.data);
+        //   })
+        //   .catch((err) => {
+        //     alert("Server error, Try again");
+        //     console.log(err);
+        //   });
```

Comment:
Are mentors stats not available from backend yet? You commented out the old request and also the new code you were writing.